archivers/7zip/Portfile

```diff
@@ -10,7 +10,7 @@ legacysupport.newest_darwin_requires_legacy 16
 
 name                7zip
 version             22.00
-revision            0
+revision            1
```

Comment:
If this is purely a fix for build failures on 10.13 and earlier, there's no need for a rev-bump.